apps/web/src/views/Ifos/hooks/v2/useGetPublicIfoData.ts

```diff
@@ -66,7 +66,7 @@ const useGetPublicIfoData = (ifo: Ifo): PublicIfoData => {
 
   const fetchIfoData = useCallback(
     async (currentBlock: number) => {
-      const client = publicClient({ chainId: ChainId.BSC })
+      const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```